aiidalab_widgets_base/computational_resources.py

```diff
@@ -902,16 +936,15 @@ def on_setup_computer(self, _=None):
         )
         try:
             computer = computer_builder.new()
-            self._configure_computer(computer)
+            self._configure_computer(computer, self.transport.value)
+            computer.store()
```

Comment:
I've moved this here since it can also fail.

Strangely, when a user provides empty workdir, the store command raises ValidationError, but the computer is still stored. That to me seems like `aiida-core` bug.